packages/storybook/stories/components/textarea/textarea.stories.ts

```diff
@@ -142,6 +142,14 @@ export const Demo: StoryObj = {
       },
       control: 'boolean',
     },
+    isRequired: {
```

Comment:
rebase and this should be done, check that you don't end up with a double `isRequired` as it did happen for me before